api/templates/api.go

```diff
@@ -9,4 +9,4 @@ import (
 //go:embed gen/openapi.yaml
 //go:embed gen/openapi3.json
 //go:embed gen/openapi3.yaml
-var ApiDocsFS embed.FS
+var DocsFS embed.FS
```

Comment:
nazov premennej/funkcie by nemal zacinat nazvom balicka,
potom to finalne vyzera ako` api.ApiDocsFs`, staci `api.DocsFs`.